### DISCUSSION TITLE
Loss Function Error!!!

### DISCUSSION THREAD

**Discussion by ppandit95** [lululxvi/deepxde#838] [Q&A]
Dear Dr Lu

I am working with a diffusion problem which involves learning the inter diffusivity coefficients which are functions of composition and constraints on these diffusivity coefficients for which DeepXDE provides excellent opportunity to pose an inverse problem in order to get these coefficients from constrained problem for which I am using the hPINN technique but after coding the  problem I came across an error as -
ValueError: ('Could not interpret loss function identifier:', <deepxde.icbc.boundary_conditions.PointSetBC object at 0x7f95407a8700>)

It will be helpful if you could comment on this error so that I can try to understand the error in a better way.

Many Thanks
Pushkar

**Comment by lululxvi** [comment]
I am not sure where the error comes from.
